docs/API/Elements.md

```diff
@@ -5,18 +5,60 @@ title: Elements
 
 The Elements API is a powerful abstraction that allows users to manipulate any property on any element through a handful of simple methods. This is important because these few methods allow for easy networking or persistent state. 
 
+To change these properties you will need to reference an object. The most common form to reference an object would be:
+
+```js
+var a = this; 
+```
+You can then use this reference to change other properties.
+```js
+var a = this; 
+a.visible = true; // set object visible to on
+```
+
+### Common properties
+
+Many functions use ID's.
```

Comment:
No need for an apostrophe here, just use "IDs." Also might be good to add a colon afterwards too, i.e. "IDs:"